src/Employer/Employer.Web/appsettings.Development.json

```diff
@@ -22,7 +22,7 @@
     "EmployerIdamsSiteUrl": "https://test-login.apprenticeships.education.gov.uk"
   },
   "Features": {
-    "AllowThroughFaaApplicationMethod": true
+    "CanShowVacancyAnalytics": true

```

Comment:
Features should be a noun, `VacancyAnalytics`

Reply:
As discussed, you have raised a valid point but I will not change this as our "features" are boolean based at the moment. There needs to be clarification around what "features" are rather than integration or behavioural switches and actual feature toggles and where FeatureNames need to be declared for coupling reasons but also for consolidation reasons. Will create a Confluence page.